partner_event/tests/test_event_registration.py

```diff
@@ -75,7 +75,7 @@ def test_button_register(self):
         event_1 = self.event_0.copy()
         wizard = self.env["res.partner.register.event"].create({"event": event_1.id})
         active_ids = [self.partner_01.id, self.registration_02.attendee_partner_id.id]
-        wizard.with_context({"active_ids": active_ids}).button_register()
+        wizard.with_context(**{"active_ids": active_ids}).button_register()
```

Comment:
It feels more natural to do it with key arguments instead of a dict unpack when the number is just one or two
```suggestion
        wizard.with_context(active_ids=active_ids).button_register()

Reply:
Thanks for the proposal, I will take it into account for the next one.